### PR TITLE
Feature/story map

### PR DESCRIPTION
**Pull request by pierreleripoll**
# Describe your changes

Added a story map, meaning a list of section components with HTML & vue i18n english/french texts. This was created with the PPTX from Emmanuel Dubois

# Related GitHub issues and pull requests

- Ref: #1

### CODE REVIEW

File: frontend/src/components/MapLibreMap.vue [hunk 1]
```diff
@@ -64,27 +73,32 @@ let draw: MapboxDraw | undefined = undefined;
...
-  );
+
+  if (props.extended) {
+    map.addControl(new GeolocateControl({}));
+    map.addControl(new FullscreenControl({}));
+    map.addControl(draw as unknown as IControl);
+    map.addControl(
+      new MaplibreGeocoder(geocoderApi, {
+        maplibregl: { Marker },
+        showResultsWhileTyping: true,
```
Comment (ymarcon) [inline frontend/src/components/MapLibreMap.vue:97]: You could specify the current locale, see example in 
https://github.com/EPFL-ENAC/alice-perl-eerl-urbtrees/blob/main/src/components/MapLibreMap.vue#L98

**Review by ymarcon — COMMENTED** [review]
The story page layout is good for a computer screen but for a tablet/mobile screen there are a lot of scrolls in every directions.

**Comment by ymarcon** [comment]
You could set different consecutive background colors for the story pages so that they can be distinguished when going through the story.

**Comment by pierreleripoll** [comment]
@ymarcon I modified responsive & added alternate background, is it good for you ? Since my feature doesn't imply modifying MapLibreMap component I suggest we keep the current locale improvement for another branch ?

**Review by ymarcon — APPROVED** [review]
Much better :+1:

**Pull request merged**